### PR TITLE
[IMP] account_invoice_consolidated

### PR DESCRIPTION
**Pull request by osi-scampbell**
The following PR changes consolidated invoices to fit a multi-company setup

### CODE REVIEW

**Comment by osi-scampbell** [comment]
@patrickrwilson The Travis and Runbot are green, the other 2 are testing code coverage. @max3903 I will write a test case for this module now.

**Comment by max3903** [comment]
Ping @osi-scampbell

File: account_invoice_consolidated/tests/test_consolidated_invoices_data.xml [hunk 1]
```diff
@@ -0,0 +1,386 @@
+
```
Comment (max3903) [inline account_invoice_consolidated/tests/test_consolidated_invoices_data.xml]:
```suggestion
```

**Review by max3903 — APPROVED** [review]

**Review by patrickrwilson — APPROVED** [review]

**Pull request merged**